doc/howto/network_forwards.md

```diff
@@ -43,9 +43,14 @@ Example:
 lxc network forward list lxdbr0
 ```
 
-## View a network forward
+```{note}
+This list displays the listen address of the network forward and its default target address, if set. To view a network forward's target addresses and ports {ref}`set in its port specifications <network-forwards-port-specifications>`, you can {ref}`show details about the network forward <network-forward-show>` or {ref}`edit the network forward <network-forward-edit>`.
```

Comment:
Suggest instead of

> To view a network forward's target addresses...

We use this

> To view the target addresses for a network forward's ports...